### PR TITLE
update Private Cluster page content

### PR DESCRIPTION
**Pull request by AlekseyKorzik**

### CODE REVIEW

File: docs/docs/setting-up/networking-instructions/private-cluster.md [hunk 1]
```diff
@@ -40,18 +52,21 @@ export BACALHAU_API_PORT=1234
...
-bacalhau serve --private-internal-ipfs --peer /ip4/<ip-address>/tcp/1235/p2p/<peer-id> --ipfs-swarm-addr /ip4/<ip-address>/tcp/<port>/p2p/<peer-id>
+bacalhau serve \
+--private-internal-ipfs \
+--peer /ip4/<ip-address>/tcp/1235/p2p/<peer-id> \
+--ipfs-swarm-addr /ip4/<ip-address>/tcp/<port>/p2p/<peer-id>
 ```
 
 :::tip
-The exact command will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
+The exact command arguments will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/private-cluster.md:62]:
```suggestion
The exact command arguments will be different on each computer and is output by the `bacalhau serve --node-type requester ...` command
```
I realise outputted was there before but it's the wrong tense.

File: docs/docs/setting-up/networking-instructions/private-cluster.md [hunk 2]
```diff
@@ -40,18 +52,21 @@ export BACALHAU_API_PORT=1234
...
 :::
 
 The command `bacalhau serve --private-internal-ipfs --peer ...` starts up a compute node and adds it to the cluster.
 
 ## Submitting Jobs
 
-To use this cluster from the client, run the following commands in your shell:
+To use this cluster from the client, run the following commands in your shell, setting following environment variables:
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/private-cluster.md:69]:
```suggestion
To use this cluster from the client, run the following commands in your shell, setting the following environment variables:
```

File: docs/docs/setting-up/networking-instructions/private-cluster.md [hunk 3]
```diff
@@ -60,10 +75,13 @@ export BACALHAU_API_PORT=1234
 ```
 
 :::tip
-The exact command will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
+The exact command arguments will be different on each computer and is outputted by the `bacalhau serve --node-type requester ...` command
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/private-cluster.md:78]:
```suggestion
The exact command arguments will be different on each computer and is output by the `bacalhau serve --node-type requester ...` command
```

File: docs/docs/setting-up/networking-instructions/private-cluster.md [hunk 4]
```diff
@@ -60,10 +75,13 @@ export BACALHAU_API_PORT=1234
...
 :::
 
-The command `export BACALHAU_IPFS_SWARM_ADDRESSES=...` sends jobs into the cluster from the command line client.
+The command `export BACALHAU_IPFS_SWARM_ADDRESSES=...` sets environment variable, which overrides default ipfs swarm address, allowing to send jobs into the cluster from the command line client. You can also use `bacalhau config set` with a `node.ipfs.swarmaddresses` parameter to change the ipfs swarm address:
```
Comment (rossjones) [inline docs/docs/setting-up/networking-instructions/private-cluster.md:81]:
```suggestion
The command `export BACALHAU_IPFS_SWARM_ADDRESSES=...` sets an environment variable which overrides the default ipfs swarm address, allowing the command line client to interact with the cluster. You can also use `bacalhau config set` with a `node.ipfs.swarmaddresses` parameter to change the ipfs swarm address:
```

File: docs/docs/setting-up/networking-instructions/private-cluster.md (hunk 1 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/private-cluster.md]: fixed

File: docs/docs/setting-up/networking-instructions/private-cluster.md (hunk 2 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/private-cluster.md]: fixed

File: docs/docs/setting-up/networking-instructions/private-cluster.md (hunk 3 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/private-cluster.md]: fixed

File: docs/docs/setting-up/networking-instructions/private-cluster.md (hunk 4 above)
Comment (AlekseyKorzik) [reply docs/docs/setting-up/networking-instructions/private-cluster.md]: fixed

**Review by rossjones — APPROVED** [review]

**Pull request merged**